docs/browser-support

```diff
@@ -0,0 +1,49 @@
+.. This Source Code Form is subject to the terms of the Mozilla Public
+.. License, v. 2.0. If a copy of the MPL was not distributed with this
+.. file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+.. _browser_support:
+
+=================
+Browser Support
+=================
+
+We seek to provide usable experiences of our most important web content to all user agents. But newer browsers are far more capable than older browsers, and the capabilities they provide are valuable to developers and site visitors. We **will** take advantage of modern browser capabilities. Older browsers **will** have a different experience of the website than newer browsers. We will strike this balance by generally adhering to the core principles of `Progressive Enhancement <https://en.wikipedia.org/wiki/Progressive_enhancement>`_::
+
+    * Basic content should be accessible to all web browsers
+    * Basic functionality should be accessible to all web browsers
+    * Sparse, semantic markup contains all content
+    * Enhanced layout is provided by externally linked CSS
+    * Enhanced behavior is provided by unobtrusive, externally linked JavaScript
+    * End-user web browser preferences are respected
+    
+Some website experiences may require us to deviate from these principles -- imagine *a marketing campaign page built under timeline pressure to deliver novel functionality to a particular locale for a short while* -- but those will be exceptions and rare.
+
+Technical details
+-------------------
+
+We deliver enhanced CSS & JS to browsers in our browser support matrix (below). We deliver basic support to all other user agents.
+
+Basic support consists of no page-specific CSS or JS. Instead, we deliver basic semantic HTML, a universal CSS stylesheet that gets applied to all pages, and a universal JS bundle that only handles downloading Firefox (click a button, get a file), and Google Analytics. 
+
+Browser Support Matrix (Updated 20190809)
+-------------------
+
+**The following browsers have enhanced support:**
+
+  * All evergreen browsers (Firefox, Chrome, Safari, Edge, Opera, etc.)
+  * IE11.
+
+**The following browsers have basic support:**
+
+  * All other user agents.
```

Comment:
Nit: suggest changing this to `All other IE browsers.`, since that is the only user agent to support conditional comments, which is the mechanism we currently use to deliver basic support styles.